MyChart.js

```diff
@@ -0,0 +1,154 @@
+const sites = require('../data/sites.json');
```

Comment:
I don't get why this file is named MyChart but the file indicates that it's for Lynn Tech?

Reply:
I changed the error line referring to Lynn Tech. Refactoring error.